src/main/scala/beam/agentsim/scheduler/BeamAgentScheduler.scala

```diff
@@ -105,43 +113,14 @@ class BeamAgentScheduler(val beamConfig: BeamConfig, stopTick: Double, val maxWi
   private var startSender: ActorRef = _
   private var nowInSeconds: Double = 0.0
 
-  private var previousTotalAwaitingRespone = 0L
-  private var currentTotalAwaitingResponse = 0L
-  private var numberRepeats = 0
-
   // Event stream state and cleanup management
   private var currentIter: Int = -1
   private val eventSubscriberRef = context.system.actorSelection(context.system./(SUBSCRIBER_NAME))
 
-  private val monitorTask =
-    if (beamConfig.beam.debug.debugEnabled)
-      Some(
-        context.system.scheduler.schedule(
-          new FiniteDuration(1, TimeUnit.SECONDS),
-          new FiniteDuration(3, TimeUnit.SECONDS),
-          self,
-          Monitor
-        )
-      )
-    else None
-  private val skipOverBadActorsTask =
-    if (beamConfig.beam.debug.skipOverBadActors)
-      Some(
-        context.system.scheduler.schedule(
-          new FiniteDuration(beamConfig.beam.debug.secondsToWaitForSkip * 2, TimeUnit.SECONDS),
-          new FiniteDuration(
-            math.round(beamConfig.beam.debug.secondsToWaitForSkip / 4.0),
-            TimeUnit.SECONDS
-          ),
-          self,
-          SkipOverBadActors
-        )
-      )
-    else None
+  private val stToStuckTimes: mutable.HashMap[ScheduledTrigger, Int] = mutable.HashMap.empty
```

Comment:
This is understandable only when you see the type. Otherwise it isn't easily understood and should be `scheduledTriggerToStuckTimes` or even just `triggerToStuckTimes`